src/graph/GraphStats.h

```diff
@@ -0,0 +1,73 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#ifndef GRAPH_GRAPHSTATS_H
+#define GRAPH_GRAPHSTATS_H
+
+#include "stats/StatsManager.h"
+#include "stats/Stats.h"
+
+namespace nebula {
+namespace graph {
+
+class GraphStats final {
+public:
+    GraphStats() {
+        storageClientStats_ = std::make_unique<stats::Stats>("storageCliet");
+        metaClientStats_ = std::make_unique<stats::Stats>("metaClient");
+        graphStats_ = std::make_unique<stats::Stats>("graph");
+        parseErrorStats_ = std::make_unique<stats::Stats>("parseError");
+        insertVStats_ = std::make_unique<stats::Stats>("insertVertex");
+        insertEStats_ = std::make_unique<stats::Stats>("insertEdge");
+        deleteVStats_ = std::make_unique<stats::Stats>("deleteVertex");
+        deleteEStats_ = std::make_unique<stats::Stats>("deleteEdge");
+        updateVStats_ = std::make_unique<stats::Stats>("updateVertex");
+        updateEStats_ = std::make_unique<stats::Stats>("updateEdge");
+        goStats_ = std::make_unique<stats::Stats>("go");
+        findPathStats_ = std::make_unique<stats::Stats>("findPath");
+        fetchVStats_ = std::make_unique<stats::Stats>("fetchVertex");
+        fetchEStats_ = std::make_unique<stats::Stats>("fetchEdge");
```

Comment:
We should separate graph, meta and storage metrics in some way like `graph_` prefix. Otherwise,  some metrics with same name maybe export into prometheus at same time.

Reply:
When you fetch the data, you know whether it's graphd, metad, or storaged